webapp-django/crashstats/supersearch/templates/supersearch/search_results.html

```diff
@@ -54,9 +45,20 @@
                         {% endif %}
                             title="Click to leave this search and go to this signature report"
                         >
-                            Signature report
+                            {{ crash[column] }}
+                        </a>
+                        {% else %}
+                        <a href="{{ current_url }}&amp;{{ column }}=%3D{{ crash[column] }}" class="term" data-field="{{ column }}" data-content="{{ crash[column] }}" title="Add this term to the search form">
+                            {{ crash[column] }}
                         </a>
-                    </em>
+                        {% endif %}
+                    {% endif %}
+
+                    {% if column == 'signature' %}
+                    &nbsp;
+                    <a href="{{ current_url }}&amp;{{ column }}=%3D{{ crash[column] }}" class="term" data-field="{{ column }}" data-content="{{ crash[column] }}" title="Add this term to the search form">
+                        Add term
+                    </a>
                     {% endif %}
```

Comment:
I would put this block one level up (inside the ``if crash[column]`` block) so that it doesn't show this link for empty signatures. Signatures are never supposed to be empty for us, but it might be for other users.